research/schooltests/sql.txt

```diff
@@ -0,0 +1,126 @@
+
+-------------------------------
+
+Füge folgende Daten in die Genre-Tabelle ein: 
+
+bezeichnung: Rock
+beschreibung: Rockmusik, kurz auch Rock, dient als Oberbegriff für Musikrichtungen, die sich seit Ende der 1960er Jahre aus der Vermischung des Rock ’n’ Roll der späten 1950er und frühen 1960er Jahre. 
+
+INSERT INTO Genre (bezeichnung, beschreibung) 
```

Comment:
Just to be safe: "Rockbeschreibung" is replaced by the actual description in the opal test?

Reply:
already discussed - its "rockige Musik" now